contrib/nydusify/cmd/nydusify.go

```diff
@@ -425,6 +425,13 @@ func main() {
 					Usage:   "File path to save the metrics collected during conversion in JSON format, for example: './output.json'",
 					EnvVars: []string{"OUTPUT_JSON"},
 				},
+				&cli.StringSliceFlag{
```

Comment:
[smoke test](https://github.com/dragonflyoss/nydus/tree/master/smoke) is required. Please also update doc